FrontEnd/screens/GameOver.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import { 
+  StyleSheet, 
+  Text, View, 
+  TouchableOpacity,
+  ImageBackground,
+  KeyboardAvoidingView
+
+ } from 'react-native';
+import { Button } from 'react-native-elements';
+import Home from './home';
+import { Actions } from 'react-native-router-flux';
+
+
+export default class Form extends Component {
```

Comment:
Since there's already a component called `Form`, can you name this `GameOverScreen` or something of the sort? And then make sure to change that name in `App.js`